### PR TITLE
Week2 consolidated

### PR DESCRIPTION
**Pull request by Shanb007**

### CODE REVIEW

File: src/com/tekion/game/models/Inning.java [hunk 1]
```diff
@@ -1,85 +1,235 @@
...
-    private void scoreBoardFirstInnings(){
-
+    public void InitiateSecondInnings(Team team2, Team team1, int overs){
+        secondInnings(team2,team1, overs);
     }
 
-    static void secondInnings(Team SecondInningsTeam , Team FirstInningsTeam, int overs){
+    private void secondInnings(Team SecondInningsTeam, Team FirstInningsTeam, int overs){
+        int currentOnStrike = 0;
+        int currentNotOnStrike = 1;
```
Comment (AdityaVikramDalmia) [inline src/com/tekion/game/models/Inning.java]: long method

File: src/com/tekion/game/models/Strike.java [hunk 2]
```diff
@@ -0,0 +1,22 @@
+package com.tekion.game.models;
+
+public class Strike {
+private int onStrike = 0;
```
Comment (AdityaVikramDalmia) [inline src/com/tekion/game/models/Strike.java:4]: cant this be an enum

File: src/com/tekion/game/models/Inning.java (hunk 1 above)
Comment (Shanb007) [reply src/com/tekion/game/models/Inning.java]: fixed

Comment (Shanb007) [reply src/com/tekion/game/models/Inning.java]: > long method

fixed

**Pull request merged**